vizro-ai/docs/pages/user_guides/run_vizro_ai.md

```diff
@@ -80,3 +80,18 @@ Vizro-AI's `_get_chart_code` method returns the Python code string that can be u
 
 The returned `code_string` can be used to dynamically render charts within your application. You may have the option to encapsulate the chart within a `fig` object or convert the figure into a JSON string for further integration.
 In case you would like to use the insights or code explanation, you can use `vizro_ai._run_plot_tasks(df, ..., explain=True)`, which returns a dictionary containing the code explanation and chart insights alongside the code.
+
+### 4. `max_debug_retry` Parameter in plot Function
```

Comment:
```suggestion
### 4. `max_debug_retry` parameter in plot function
```

Reply:
this is follow all title formatting in this page

Reply:
You mean like `Application integration`? Hm not sure, I found it looked a little strange, but happy to keep it!

Reply:
I thought it was always just the first letter :P

Reply:
updated all of them to be just first letter now